react/ProductSearchContextProvider.js

```diff
@@ -25,6 +25,8 @@ class ProductSearchContextProvider extends Component {
     /** Render runtime context */
     runtime: PropTypes.shape({
       page: PropTypes.string.isRequired,
+      prefetchPage: PropTypes.func.isRequired,
```

Comment:
Is it necessary?

Reply:
yeah, eslint was complaining because it was missing and was being used (@luciannojunior forgot to add it in his pr)